llvm/lib/SYCLLowerIR/ESIMD/LowerESIMD.cpp

```diff
@@ -494,6 +494,7 @@ class ESIMDIntrinDescTable {
         {"dpasw", {"dpasw", {a(0), a(1), a(2), t(0)}}},
         {"dpasw_nosrc0", {"dpasw.nosrc0", {a(0), a(1), t(0)}}},
         {"nbarrier", {"nbarrier", {a(0), a(1), a(2)}}},
+        {"nbarrier_arrive", {"nbarrier.arrive", {a(0), a(1), a(2), a(3)}}},
         {"raw_send_nbarrier_signal",
```

Comment:
Can you please also remove this line with 'raw_send_nbarrier_signal'

Reply:
fixed in https://github.com/intel/llvm/pull/12982/commits/7fa62a1e0a2b8d81b4fcc4c6f45861820cbfeb7b, thanks